roles/storage-cns/tasks/provision.yml

```diff
@@ -5,6 +5,15 @@
         and admin_user is defined
         and admin_password is defined
 
+- name: Determine the heketi route
+  command: oc get route heketi-storage -n app-storage -o jsonpath="{.spec.host}"
```

Comment:
If you're doing this because you can't assume the URL to heketi, you can't assume the name of the route being `heketi-storage`, nor the namespace. Can you make these parameters and then set these as default values?

Reply:
Good point.  Will do.